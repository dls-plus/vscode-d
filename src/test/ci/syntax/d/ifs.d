void main() {
	if (x) {
	}
	if (x)
	{}

	while (x) {}
	for (x; y; z) {}
	foreach (x, y; z) {}
	switch (x) {}
	final switch (x) {}
}