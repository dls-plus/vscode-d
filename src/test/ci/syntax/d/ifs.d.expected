[{"start":0,"end":4,"scope":"storage.type.basic-type.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":9,"scope":"entity.name.function.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":11,"scope":"source.d"},{"start":11,"end":14,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":3,"scope":"keyword.control.if.d"},{"start":3,"end":4,"scope":"source.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"variable.d"},{"start":6,"end":7,"scope":"source.d"},{"start":7,"end":10,"scope":"source.d"}]
[{"start":0,"end":3,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":3,"scope":"keyword.control.if.d"},{"start":3,"end":4,"scope":"source.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"variable.d"},{"start":6,"end":7,"scope":"source.d"}]
[{"start":0,"end":4,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":6,"scope":"keyword.control.while.d"},{"start":6,"end":7,"scope":"source.d"},{"start":7,"end":8,"scope":"source.d"},{"start":8,"end":9,"scope":"variable.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":14,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":4,"scope":"keyword.control.for.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"source.d"},{"start":6,"end":7,"scope":"variable.d"},{"start":7,"end":8,"scope":"keyword.operator.semi-colon.d"},{"start":8,"end":9,"scope":"source.d"},{"start":9,"end":10,"scope":"variable.d"},{"start":10,"end":11,"scope":"keyword.operator.semi-colon.d"},{"start":11,"end":12,"scope":"source.d"},{"start":12,"end":13,"scope":"variable.d"},{"start":13,"end":14,"scope":"source.d"},{"start":14,"end":18,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":8,"scope":"keyword.control.foreach.d"},{"start":8,"end":9,"scope":"source.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":11,"scope":"variable.d"},{"start":11,"end":12,"scope":"keyword.operator.comma.d"},{"start":12,"end":13,"scope":"source.d"},{"start":13,"end":14,"scope":"variable.d"},{"start":14,"end":15,"scope":"keyword.operator.semi-colon.d"},{"start":15,"end":16,"scope":"source.d"},{"start":16,"end":17,"scope":"variable.d"},{"start":17,"end":18,"scope":"source.d"},{"start":18,"end":22,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":7,"scope":"keyword.control.switch.d"},{"start":7,"end":8,"scope":"source.d"},{"start":8,"end":9,"scope":"source.d"},{"start":9,"end":10,"scope":"variable.d"},{"start":10,"end":11,"scope":"source.d"},{"start":11,"end":15,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":13,"scope":"keyword.control.final.switch.d"},{"start":13,"end":14,"scope":"source.d"},{"start":14,"end":15,"scope":"source.d"},{"start":15,"end":16,"scope":"variable.d"},{"start":16,"end":17,"scope":"source.d"},{"start":17,"end":21,"scope":"source.d"}]
[{"start":0,"end":2,"scope":"source.d"}]